src/main/java/seedu/address/logic/wanderlustlogic/LogicManager.java

```diff
@@ -94,6 +94,7 @@ public Directory getDirectory() {
         return model.getDirectory();
     }
 
+    // TODO:
```

Comment:
Just clarifying here, what's there TODO here?

Reply:
oh cos this method is I cincai add one last time. No JavaDoc all that, need to redo